scripts/inactive_issue_checker_test.py

```diff
@@ -597,7 +598,7 @@ def test_add_alert_comment_on_issue(
         self.service.add_alert_comment_on_issue(issue)
 
         expected_comment = (
-            f'Hi @{issue.assignee_username} this PR is inactive '
+            f'Hi @user1 this PR is inactive '
             f'for {INACTIVE_DAYS_THRESHOLD} and you will be '
```

Comment:
You still have {INACTIVE_DAYS_THRESHOLD} as a variable. If you get a general comment please fix it everywhere, not only on the instance mentioned. Please check your other expected values in the tests, too.